woocommerce/woocommerce
# [CYS] Go to the assembler when the cys task is completed

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the "Design you own" button in the `Extensions > Themes` section to go to the assembler if the theme was already customized.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Go to `WooCommerce > Home`, click on `Start customizing`, finish the flow and click on `Done` to finish the CYS task.
4. Go to `wp-admin/admin.php?page=wc-admin&tab=themes&path=%2Fextensions` and click on the `Design your own` button and make sure you get redirected to the assembler and the changes you did to the theme are preserved.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
CYS - Go to the assembler when clicking to the "Design you own" button if the theme was already customized in the assembler.
#### Comment 

</details>
